### ISSUE TITLE
... missing field `live`    

### ISSUE THREAD

**Issue by DigibitDK** [715209/nginx-obs-automatic-low-bitrate-switching#89]
Noalbs v2 do not seem to know what to look for ... or am I doing something wrong?
(My v1 works perfect)

LOG:
`2022-01-26T12:16:36.239171Z ERROR NOALBS{user=testuser}:Switcher: noalbs::stream_servers::nginx: Error parsing stats (http://172.20.0.2:8085/stats) missing field `live` 
`
Part of config:
```
    },
    "switchingScenes": {
      "normal": "rtmp",
      "low": "rtmp_low",
      "offline": "rtmp_off"
    },
    "streamServers": [
      {
        "streamServer": {
          "type": "Nginx",
          "statsUrl": "http://172.20.0.2:8085/stats",
          "application": "test_1234",
          "key": "testuser"
        },
        "name": "RTMP",
        "priority": 0,
        "overrideScenes": null,
        "dependsOn": null
      }
]
```

Some other things ...
Config-conversion from v1 to v2, do not convert old scene-names. (My scenes are renamed compared to original v1-config names)
Does v2 not use 'refresh-scene' anymore? Can only find "normal", "low" and "offline".

NOALBS 2.4.2 // RTMP NGINX

**Comment by 715209** [comment]
Hi DigibitDK, 

Thanks for reporting these issues. 

I am unable to reproduce the NGINX parsing issue, could you please send me the raw XML output of the stats page. You can do this by going to the stats page and viewing the page source.

I have fixed the config conversion to work as expected, it will be available in the next release.
NOALBS v2 does not use the refresh scene anymore since the fix command should solve all issues.

**Comment by DigibitDK** [comment]
Hi Brian. 
How can i send it to you? I can't publish it here for security reasons. (Im a RTMP-provider)

If fix works the exact same way as in v1, then i'm a little concerned about the removing of the refresh-function, since fix not always seems to do the trick. OBS do strange things from time to time, that fix do not seem to recover. But that function might be diffrent in v2, I need to test that.

**Comment by DigibitDK** [comment]
(Discord friend request sent)

**Comment by DigibitDK** [comment]
Af output of the RTMP-log. v1 vs v2. Not sure if that gives any clues. (With no incoming stream)
v1:
172.20.1.12 - - [27/Jan/2022:15:03:40 +0000] "GET /stats HTTP/1.1" 200 9327 "-" "-"

v2:
172.20.1.11 - - [27/Jan/2022:15:04:52 +0000] "GET /stats HTTP/1.1" 200 9326 "-" "node-fetch/1.0 (+https://github.com/bitinn/node-fetch)"

**Comment by DigibitDK** [comment]
Issues confirmed fixed in 2.5.1
